presto-main/src/test/java/io/prestosql/spiller/TestFileSingleStreamSpillerFactory.java

```diff
@@ -52,6 +56,7 @@
     private ListeningExecutorService executor;
     private File spillPath1;
     private File spillPath2;
+    private FileSingleStreamSpillerFactory spillerFactory;
```

Comment:
instead of extracting variable I would extract factory method. Also it is a good practice to have refactoring changes in separate commits from actual logic changes. That way it is easier to review and to read the git history.